Adamant/Modules/Wallets/Ethereum/EthWalletService+RichMessageProviderWithStatusCheck.swift

```diff
@@ -71,7 +71,7 @@ private extension EthWalletService {
     func getTransactionInfo(hash: String, web3: Web3) async throws -> EthTransactionInfo {
         try await withThrowingTaskGroup(
             of: EthTransactionInfoElement.self,
-            returning: EthTransactionInfo.self
```

Comment:
Yes, that's too much, but let's keep it in the 3.3.0 release to guarantee that a crash will not appear hear. Most ETH crashes appeared in this place